docs/user/dql/functions.rst

```diff
@@ -1425,6 +1425,54 @@ Example::
     | 26                               |
     +----------------------------------+
 
+DAY_OF_MONTH
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: day_of_month(date) extracts the day of the month for date, in the range 1 to 31. The dates with value 0 such as '0000-00-00' or '2008-00-00' are invalid.
```

Comment:
nit: This seems obvious, but I see that it was included in the synonym functions... 
If you'd like, we could remove it from all of the functions.  
```
The dates with value 0 such as '0000-00-00' or '2008-00-00' are invalid.
```

Reply:
I agree it's kind of obvious. I will remove it for now and in my PR upstream just add it back in if Amazon wants it back.

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/194/commits/878a6a152955d3cbbc1a21c85ff2d900a6bc785c